src/devices/tuya.ts

```diff
@@ -6101,6 +6101,85 @@ const definitions: Definition[] = [
             ],
         },
     },
+	{
+		fingerprint: [{
+			modelID: 'TS0601',
+			manufacturerName: '_TZE204_e9ajs4ft',
+		}],
+		model: 'CTL-R1-TY-Zigbee',
+		vendor: 'TuYa',
+		description: 'Creatrol - 24G radar human presence motion sensor.',
+		fromZigbee: [tuya.fz.datapoints],
+		toZigbee: [tuya.tz.datapoints],
+		exposes: [
+			e.illuminance_lux(),
+			e.presence(),
+			e.numeric('presence_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(100)
+			.withValueStep(1).withUnit('%').withDescription('presence_sensitivity'),
+	
+			e.numeric('detection_range', ea.STATE_SET).withValueMin(1.5).withValueMax(4.5).withValueStep(
+				0.1).withUnit('m').withDescription('Detection range'),
+	
+			e.numeric('detection_delay', ea.STATE_SET).withValueMin(1).withValueMax(600).withValueStep(1)
+			.withUnit('s').withDescription('presence detection delay'),
+	
+			e.numeric('illumin_max', ea.STATE_SET).withValueMin(0).withValueMax(2000).withValueStep(1)
+			.withUnit('Lux').withDescription(
+				'The threshold of the light sensor will be linked to the "light_switch" parameter'),
+	
+			e.numeric('illumin_min', ea.STATE_SET).withValueMin(0).withValueMax(1000).withValueStep(1)
+			.withUnit('Lux').withDescription(
+				'The threshold of the light sensor will be linked to the "light_switch" parameter'),
+			e.enum('presence_illumin_switch', ea.STATE_SET, ['disable', 'enable']).withDescription(
+				'Whether to enable illumination threshold detection'),
+			e.binary('light_switch', ea.STATE, "on", "off").withDescription(
+				'This state will determine the light on/off based on the lighting threshold and presence sensing'
+				),
+			e.enum('light_linkage', ea.STATE_SET, ['disable', 'enable']).withDescription('linkage_light'),
+			e.enum('detection_method', ea.STATE_SET, ['only move', 'exist move']).withDescription(
+				'detection method'),
+			e.enum('backlight_mode', ea.STATE_SET, ['open', 'close', 'lighting']).withDescription(
+				'backlight mode'),
+			e.binary('find_switch', ea.STATE_SET, "on", "off").withDescription(
```

Comment:
Does the indicator light stay on or does it turn off automatically?

Reply:
When "find_switch" is set to "on", the indicator light will rapidly blink, making it easier for debugging personnel to locate the product's position.